app/src/main/java/com/example/KnowYouLimit/ui/main/MainViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.KnowYouLimit.ui.main
+
+import androidx.lifecycle.MutableLiveData
+import com.example.KnowYouLimit.dto.drinks
+import com.example.KnowYouLimit.service.DrinksService
+
+class MainViewModel {
+    var drinks: MutableLiveData<ArrayList<drinks>> = MutableLiveData()
+    var drinksService : DrinksService = DrinksService()
+
+    fun fetchDrinks(name: String){
+        drinks = drinksService.fetchDrinks(name)
+    }
+}
```

Comment:
Allowing fetchDrinks to be accessible in the main view model from the service that was implemented.